code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -1,14 +1,11 @@
 /datum/game_mode/traitor/double_agents
 	name = "double agents"
 	config_tag = "double_agents"
-	restricted_jobs = list("Cyborg", "AI", "Captain", "Head of Personnel", "Chief Medical Officer", "Research Director", "Chief Engineer", "Head of Security") // Human / Minor roles only.

```

Comment:
No because silicons are counted as dead automatically and has to be this way by current design. You can technically kill your target borg em and emag.